src/app/tl-drop-down/tl-drop-down.component.html

```diff
@@ -11,10 +11,30 @@
   [menuXOffset]="'var(--tl-dropdown-menu-x-offset)'"
   [menuYOffset]="'var(--tl-dropdown-menu-y-offset)'"
   [menuItemMargin]="'var(--tl-dropdown-menu-item-margin)'"
->
+  [btnBackground]="btnBackground"
+  [backgroundActive]="btnBackgroundActive"
+  [btnBackgroundHover]="btnBackgroundHover"
+  [btnBorderColor]="btnBorderColor"
+  [btnBorderActive]="btnBorderActive"
+  [btnBorderFocus]="btnBorderFocus"
+  [btnBorderHover]="btnBorderHover"
+  [btnBorderRadius]="'var(--tl-btn-border-radius)'"
+  [btnBoxShadowFocus]="'var(--tl-btn-focus-box-shadow)'"
+  [btnColor]="btnColor"
+  [btnColorActive]="btnColorActive"
+  [btnColorHover]="btnColorHover"
+  [btnFontFamily]="'var(--tl-btn-font-family)'"
+  [btnFontSize]="'var(--tl-btn-font-size)'"
+  [btnLineHeight]="'var(--tl-btn-line-height)'"
+  [btnTextAlign]="'var(--tl-btn-text-align)'"
+  [btnTextDecoration]="btnTextDecoration"
+  [btnTextDecorationActive]="btnTextDecorationActive"
+  [btnTextDecorationFocus]="btnTextDecorationFocus"
+  [btnTextDecorationHover]="btnTextDecorationHover"
+  >
   <wvr-dropdown-btn>
-    <ng-content select="tl-dropdown-btn"></ng-content>
-  </wvr-dropdown-btn >
+    <ng-content select="tl-dropdown-btn" ></ng-content>
```

Comment:
typically there are no spaces attributes and the end of the tag.

Reply:
Removed the space between the attribute and the end of the tag